pkgs/top-level/all-packages.nix

```diff
@@ -16185,6 +16185,10 @@ in
     erlang = erlangR21;
   };
 
+  couchdb3 = callPackage ../servers/http/couchdb/3.0.0.nix {
```

Comment:
shouldn't the filename be `3.nix` instead?

Reply:
Here I'd followed the 2.0.0.nix and used it as a template (relevant bellow). I kind of thought the semver made sense, but probably more for versions like 1.X and 4.X than 2 and 3, since couchdb2&couchdb3 don't have so much going on that is likely to accidentally break compatibility between their minor versions. I could rename both files 2.nix and 3.nix?

Reply:
I don't know